flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperator.java

```diff
@@ -132,6 +133,13 @@
 	 */
 	protected final OutputTag<IN> lateDataOutputTag;
 
+	/**
+	* Metrics about the lost data due to arrive late.
+	* */
+	protected final String loseData = "lost_data";
```

Comment:
we use camel case for naming metric names. "lost_data" in particular is also not really descriptive and the naming scheme is inconsistent with existing metrics.

I suggest "numLateRecords".

This string should also be static, and named `LATE_ELEMENTS_METRIC_NAME` for clarity purposes.